src/applications/vaos/components/Breadcrumbs.jsx

```diff
@@ -101,7 +101,7 @@ export default function VAOSBreadcrumbs({ children, labelOverride }) {
   return (
     <>
       <VaBreadcrumbs
-        role="navigation"
+        data-testid="vaos-breadcrumbs"
         aria-label="Breadcrumbs"
```

Comment:
based on feedback from accessibility team, we can remove the `aria-label="Breadcrumbs"`